app/jobs/whats_app_adapter/three_sixty_dialog/process_webhook_job.rb

```diff
@@ -11,7 +11,7 @@ def perform(organization_id:, components:)
         @whats_app_message = components
         adapter = WhatsAppAdapter::ThreeSixtyDialogInbound.new
 
-        adapter.consume(organization, components) { |message| message.contributor.reply(adapter) }
```

Comment:
Does this gets replaced with `@message.request = sender.active_request` from above?

Reply:
yes, and the `@message.save!`